src/Resources/config/beanstalkdui.php

```diff
@@ -31,6 +31,17 @@
 
     'middleware' => ['web', 'auth'],
 
+    /*
+    |--------------------------------------------------------------------------
+    | Route Prefix
+    |--------------------------------------------------------------------------
+    |
+    | Here you can specify the route prefix that should be applied the
+    | Admin UI routes.
+    |
+    */
+    'prefix' => '',
```

Comment:
This by itself won't be enough because the prefix does not get used in the routes that the package defines. Check out `routes.php` to see what I mean.

Reply:
You are right. But the prefix used when routes apply in the service Provider. Check https://github.com/Dionera/laravel-beanstalkd-admin-ui/blob/master/src/BeanstalkdUIServiceProvider.php.

Reply:
Now it`s fixed. Look at the /src/BeanstalkdUIServiceProvider.php